src/views/reverse/steps/payInvoice.js

```diff
@@ -46,26 +46,34 @@ const payInvoiceStyles = () => ({
 
 class StyledPayInvoice extends React.Component {
   componentDidMount() {
-    const { invoice, webln } = this.props;
+    const { swapResponse, webln } = this.props;
 
     console.log(webln);
     if (webln) {
-      webln.sendPayment(invoice);
+      webln.sendPayment(swapResponse.invoice);
     }
   }
 
   render() {
-    const { classes, asset, invoice } = this.props;
+    const { classes, swapInfo, swapResponse } = this.props;
+    const link = swapResponse
+      ? `${getExplorer(swapInfo.quote)}/${swapResponse.lockupTransactionHash}`
+      : '';
 
     return (
       <View className={classes.wrapper}>
         <View className={classes.qrcode}>
-          <QrCode size={300} link={invoice} />
+          <a href={link} target="_blank" rel="noopener noreferrer">
```

Comment:
Please rebase the branch and use the `Link` component you added with [this commit](https://github.com/BoltzExchange/boltz-frontend/commit/efcaa53ac0def6c2cd3e3c806008a0bb5313cdbf)